Source/PLCrashReportSystemInfo.m

```diff
@@ -35,6 +35,7 @@
  * The current host's operating system.
  */
 PLCrashReportOperatingSystem PLCrashReportHostOperatingSystem =
+// FIXME: Deprecated, use TARGET_OS_SIMULATOR
```

Comment:
Should we create a bug for that?

Reply:
I would rename enum value, but it's breaking change. It works, so just a comment for now